cli/cmd/helm/helmInstall.go

```diff
@@ -14,8 +14,6 @@ import (
 	"github.com/timescale/tobs/cli/pkg/utils"
```

Comment:
change file name?

Reply:
In the upcoming PR, I would like to drop `tobs helm` (these are helm subcommands) cmd, so planning to rid of these files in that PR. :)